Rubberduck.Inspections/Inspector.cs

```diff
@@ -151,6 +157,13 @@ private static bool RequiredLibrariesArePresent(IInspection inspection, Rubberdu
                 return requiredLibraries.All(library => projectNames.Contains(library.LibraryName));
             }
 
+            private static bool RequiredHostIsPresent(IInspection inspection)
+            {
+                var requiredHost = inspection.GetType().GetCustomAttribute<RequiredHostAttribute>();
+
+                return requiredHost == null || requiredHost.HostNames.Contains(Path.GetFileName(Application.ExecutablePath).ToUpper());
```

Comment:
Isn't the `Application` here a member of `Sysem.Windows.Forms`? Does it work without any winforms?

Reply:
I don't know, to be honest... Maybe @retailcoder can help here?

Reply:
Looks like it is. [The About dialog is using the same](https://github.com/rubberduck-vba/Rubberduck/blob/next/Rubberduck.Core/UI/About/AboutControlViewModel.cs#L30-L31), so it should be good.